### PR TITLE
Some premature optimization for amp-slides

### PR DESCRIPTION
**Pull request by LendsMan**
Added some optimisation if you don't mind and code style is acceptable.

### CODE REVIEW

**Comment by LendsMan** [comment]
Done.

File: extensions/amp-slides/0.1/amp-slides.js [hunk 1]
```diff
@@ -52,15 +52,16 @@ class AmpSlides extends AMP.BaseElement {
...
-    this.prevButton_.style.position = 'absolute';
-    this.prevButton_.style.zIndex = 10;
-    this.prevButton_.style.left = '16px';
-    this.prevButton_.style.top = '50%';
-    this.prevButton_.style.padding = '8px';
-    this.prevButton_.style.fontSize = '24px';
-    this.prevButton_.style.marginTop = '-20px';
-    this.prevButton_.style.pointerEvents = 'all';
+    const prevButtonStyle = this.prevButton_.style;
+    prevButtonStyle.opacity = 0.6;
```
Comment (erwinmombay) [inline extensions/amp-slides/0.1/amp-slides.js]:
```js
lets go ahead and use tr.setStyle(this.prevButton_, {
  //styles here...
});
```

it should already be imported in this file. Thanks!

File: extensions/amp-slides/0.1/amp-slides.js [hunk 2]
```diff
@@ -52,15 +52,16 @@ class AmpSlides extends AMP.BaseElement {
...
-    this.prevButton_.style.opacity = 0.6;
-    this.prevButton_.style.position = 'absolute';
-    this.prevButton_.style.zIndex = 10;
-    this.prevButton_.style.left = '16px';
-    this.prevButton_.style.top = '50%';
-    this.prevButton_.style.padding = '8px';
-    this.prevButton_.style.fontSize = '24px';
-    this.prevButton_.style.marginTop = '-20px';
-    this.prevButton_.style.pointerEvents = 'all';
+    const prevButtonStyle = this.prevButton_.style;
```
Comment (erwinmombay) [inline extensions/amp-slides/0.1/amp-slides.js]: lets go ahead and use:

```js
st.setStyles(this.prevButton_, {
  //styles here...
});
```

it should already be imported in this file. Thanks!

**Comment by erwinmombay** [comment]
@LendsMan would you mind squashing your commits into 1? let me know if you need any help on this!

**Comment by LendsMan** [comment]
I didn't pay attention to st.setStyles. Updated and squashed.

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**